schema/src/index.ts

```diff
@@ -0,0 +1,23 @@
+import express from 'express'
+var app = express()
+app.use(express.json())
+
+// define a route handler for the default home page
+app.post('/fastGen', (req: express.Request, res: express.Response) => {
+  console.log(req.body)
+  res.send(req.body)
+})
+
+// define a route handler for the default home page
+app.post('/detailedGen', (req: express.Request, res: express.Response) => {
+  console.log(req.body)
+  res.send(req.body)
+})
+
+app.get('/', (req: express.Request, res: express.Response) => {
+  res.send(200)
+})
+
+app.listen(8520, () => {

```

Comment:
We should use standard port `8080` before merging and use the routing mechanism of traefik to reach the service.